test/end-to-end.js

```diff
@@ -970,6 +976,44 @@ describe('end-to-end', function() {
       });
     });
 
+    it('shows error behavior responses in development mode', function(done) {
+      var app = loopback({ localRegistry: true, loadBuiltinModels: true });
+      var bootOptions = {
+        appRootDir: SANDBOX,
+        env: 'development'
+      };
+      boot(app, bootOptions, function(err) {
+        if (err) return done(err);
+        app.listen(0, function() {
+          request(app)
+            .get('/url-does-not-exist')
+            .end(function(err, res) {
+              expect(res.error.text.match(/url-does-not-exist/));
```

Comment:
Hmm, I find `res.error.text` a bit unexpected, I believe strong-error-handler returns different JSON properties. @davidcheung could you PTAL too?